Content.Shared/Foldable/DeployFoldableSystem.cs

```diff
@@ -59,6 +61,10 @@ private void OnAfterInteract(Entity<DeployFoldableComponent> ent, ref AfterInter
         if (args.Handled || !args.CanReach)
             return;
 
+        // Don't do anything if the target is inside a container.
+        if (args.Target.HasValue && _container.IsEntityInContainer(args.Target.Value))
+            return;
```

Comment:
```suggestion
        // Don't do anything unless you clicked on the floor.
        if (args.Target.HasValue)
            return;
```
The second check is not needed, they should simply not do anything if you use them on another entity.